packages/versioned-playground/src/VersionedSquigglePlayground.tsx

```diff
@@ -79,6 +93,11 @@ export const VersionedSquigglePlayground: FC<Props> = ({
         onCodeChange={props.onCodeChange}
         onSettingsChange={props.onSettingsChange}
         height={props.height}
+        // older playgrounds don't support these, it'll be ignored, that's fine
+        // (why TypeScript doesn't error on this, these props didn't exist in 0.8.5? no idea)
```

Comment:
yea - this seems kinda scary as we make further changes, so I kinda hope TS just plays this easy.

Reply:
It will still be possible with a huge switch statement in VersionedSquigglePlayground, so it shouldn't be a problem even if we make bigger changes (or change the semantic of some prop, e.g. a prop that's a callback with evolving signature).